recipes-rubygems/rubygems-nokogiri_1.12.3.bb

```diff
@@ -4,12 +4,13 @@ DESCRIPTION = "Nokogiri (鋸) makes it easy and painless to work with XML and HT
 HOMEPAGE = "https://nokogiri.org"
 
 LICENSE = "MIT"
-LIC_FILES_CHKSUM = "file://LICENSE-DEPENDENCIES.md;md5=2fcfc4989eb71e592d2e85d26d7fa91c"
+LIC_FILES_CHKSUM = "file://LICENSE-DEPENDENCIES.md;md5=2cf22f72d34436ea00c7c5fa20cc9242"
 
-EXTRA_DEPENDS += "\
+EXTRA_DEPENDS_append += " \
     libxml2 \
```

Comment:
potential issue: Multiple modifiers of append/prepend/remove/+= found in one operation